### ISSUE TITLE
Add media (book, audiobook, etc.) manually

### ISSUE THREAD

**Issue by Martin-Fox** [bonukai/MediaTracker#652]
It would be great if, in cases where I can't find a specific item (like an audiobook that was only released in Polish), it was possible to manually add the missing item by entering the basic information.

**Comment by mobamoba** [comment]
Yes - Openlibrary is missing tons of cover images and it would be great to be able to upload a cover myself.

**Comment by TangentFoxy** [comment]
Disappointed there's no movement on this. :\ Just started trying out this app.. and this is kind of a deal breaker because I interact with a lot of obscure media.
